index.html

```diff
@@ -17,69 +17,284 @@
+          </ul>
+        </div>
+        <button type="button" class="work-button">See Project</button>
+      </div>
+    </section>
+    <section class="about-me">
+      <div class="intro">
+        <h1>About Myself</h1>
+        <p class="mainText">
+          I’m a software developer! I can help you build a product , feature or
+          website Look through some of my work and experience! If you like what
+          you see and have a project you need coded, don’t hestiate to contact
+          me.
+        </p>
+        <div class="socialMedia">
+          <h2>LET'S CONNECT</h2>
+          <ul class="socialMediaImages">
+            <li>
+              <a href="https://twitter.com/VicPeralta">
+                <img
+                  src="./assets/twitter.svg"
+                  alt="twitter image"
+                  width="20"
+                  height="19"
+                />
+              </a>
+            </li>
+            <li>
+              <a
+                href="https://www.linkedin.com/in/victor-peralta-gomez-648072107/"
+              >
+                <img
+                  src="./assets/LinkedinIcon.svg"
+                  alt="linkedin image"
+                  width="19"
+                  height="19"
+                />
+              </a>
+            </li>
+            <li>
+              <a href="#">
+                <img
+                  src="./assets/medium.svg"
+                  alt="medium image"
+                  width="20"
+                  height="16"
+                />
+              </a>
+            </li>
+            <li>
+              <a href="https://github.com/VicPeralta">
+                <img
+                  src="./assets/github.svg"
+                  alt="github image"
+                  width="18"
+                  height="20"
+                />
+              </a>
+            </li>
+            <li>
+              <a href="#">
+                <img
+                  src="./assets/angellist.svg"
+                  alt="angellist image"
+                  width="14"
+                  height="20"
+                />
+              </a>
+            </li>
+          </ul>
+        </div>
+        <button type="button" class="work-button">Get my resume</button>
+        <div class="languages">
+          <div class="languages-option">
+            <h3 class="languages-title">Languages</h3>
+            <!-- <h3 class="languages-title">&#9660;</h3> -->
+            <button type="button" class="arrow-button arrow-down">&gt;</button>
+          </div>
+          <ul>
+            <li>
+              <img src="./assets/javascript.svg" alt="javascript image" />
+            </li>
+            <li><img src="./assets/html.svg" alt="html image" /></li>
+            <li><img src="./assets/css.svg" alt="" /></li>
+          </ul>
+        </div>
+        <div class="frameworks">
+          <ul>
+            <li>
+              <p>Frameworks</p>
+              <!-- <p>&#9658;</p> -->
```

Comment:
[OPTIONAL]

- [ ]  Please make sure to delete unnecessary comments, a well-structured and semantic HTML is "self-documenting". This will make your code cleaner.  :+1: